tests/unit/jobserver/test_auth_pipeline.py

```diff
@@ -84,7 +84,6 @@ def test_pipeline_with_new_user(slack_messages, strategy):
 
     social = output["social"]
     assert social.user == user
-    assert social.extra_data["access_token"] == "sekret"
```

Comment:
thought: Perhaps it's worth checking that `sekret` _doesn't_ appear as a value anywhere in the extra data? It's a property that we presumably wish to ensure remains so for the foreseeable future.

Reply:
Good point, that would make the test as a better expression of "we don't want to be storing this token"